web/src/engine/js-processor/src/jsKeyboardInterface.ts

```diff
@@ -390,7 +390,7 @@ export class JSKeyboardInterface extends KeyboardHarness {
    * A KMW 10+ function designed to bring KMW closer to Keyman Desktop functionality,
    * near-directly modeling (externally) the compiled form of Desktop rules' context section.
    */
-  fullContextMatch(n: number, outputTarget: OutputTarget, rule: ContextEntry[]): boolean {
+  fullContextMatch(n: number, outputTarget: OutputTargetBase, rule: ContextEntry[]): boolean {
```

Comment:
I see it's no longer `{Object} Ptarg` in the header (l. 386).

Also l. 980 and maybe a few others

Reply:
Done.